src/crab/wto.cpp

```
@@ -291,3 +291,15 @@ const wto_nesting_t& wto_t::nesting(const label_t& label) {
     }
     return _nesting.at(label);
 }
+
+// Visit the head of each loop in the WTO.
+void
+wto_t::visit_loop_heads(const std::function<void(const label_t&)>& f) const
```

cppcheck

[style] The function 'visit_loop_heads' is never used.

(unusedFunction)